### PR TITLE
[browser] rename normal scenario to `WasmTestOnV8`

### PR DESCRIPTION
**Pull request by pavelsavara**

### CODE REVIEW

File: src/libraries/sendtohelix-browser.targets [hunk 1]
```diff
@@ -177,6 +182,7 @@
   <Import Project="$(RepositoryEngineeringDir)testing\wasm-provisioning.targets" />
 
   <Target Name="PrepareHelixCorrelationPayload_Wasm">
+    <Error Condition="'$(Scenario)' == '' or '$(Scenario)' == 'normal'" Text="Only supported scenarios are WasmTestOnV8, WasmTestOnNodeJS, WasmTestOnChrome, WasmTestOnFirefox, WasmDebuggerTests and BuildWasmApps at the moment." />
```
Comment (ilonatommy) [inline src/libraries/sendtohelix-browser.targets:185]: I would prefer
```suggestion
    <Error Condition="'$(Scenario)' != 'WasmTestOnV8' and '$(Scenario)' != 'WasmTestOnNodeJS' and  '$(Scenario)' != 'WasmTestOnChrome' and '$(Scenario)' != 'WasmTestOnFirefox' and  '$(Scenario)' != 'WasmDebuggerTests' and  '$(Scenario)' != 'BuildWasmApps' Text="Only supported scenarios are WasmTestOnV8, WasmTestOnNodeJS, WasmTestOnChrome, WasmTestOnFirefox, WasmDebuggerTests and BuildWasmApps at the moment." />
```

**Review by ilonatommy — APPROVED** [review]

File: src/libraries/sendtohelix-browser.targets (hunk 1 above)
Comment (pavelsavara) [reply src/libraries/sendtohelix-browser.targets:185]: in the next PR

**Comment by pavelsavara** [comment]
/ba-b issues are https://github.com/dotnet/runtime/issues/98578

File: eng/pipelines/extra-platforms/runtime-extra-platforms-wasm.yml [hunk 2]
```diff
@@ -321,4 +321,4 @@ jobs:
       isExtraPlatformsBuild: ${{ parameters.isExtraPlatformsBuild }}
       isWasmOnlyBuild: ${{ parameters.isWasmOnlyBuild }}
       scenarios:
-        - normal
+        - WasmTestOnV8
```
Comment (maraf) [inline eng/pipelines/extra-platforms/runtime-extra-platforms-wasm.yml:324]: This is wasi

Comment (pavelsavara) [reply eng/pipelines/extra-platforms/runtime-extra-platforms-wasm.yml:324]: okay, fixed in the next PR. Thanks!!!

**Pull request merged**